### PR TITLE
Upgrade Pong Example to support game states and use player silencing

### PR DESCRIPTION
**Pull request by marc-n-dream**
Consumes the airconsole-api 1.9.0 beta at https://www.airconsole.com/api/airconsole-1.9.0-beta.js

### CODE REVIEW

File: controller.html [hunk 1]
```diff
@@ -1,103 +1,218 @@
...
+
+      var transitions = {
+        LOBBY: {
+          entryAction: () => {
+            if(screen === LOBBY) return;
+            document.getElementById("lobby").style.display = "block";
+            document.getElementById("game_mode").style.display = "none";
+            this.screen = LOBBY;
+          },
+          exitAction: () => {},
```
Comment (laurentndream) [inline controller.html]: Why do we need these exitAction?

File: screen.html [hunk 2]
```diff
@@ -9,80 +9,132 @@
...
           }
         };
-
       }
 
       /**
        * Checks if two players are connected!
        */
-      function checkTwoPlayers() {
+      function checkTwoPlayers(send = true) {
```
Comment (laurentndream) [inline screen.html:97]: are we ok with default param?

File: controller.html (hunk 1 above)
Comment (marc-n-dream) [reply controller.html]: Actually no longer, no.

File: screen.html (hunk 2 above)
Comment (marc-n-dream) [reply screen.html:97]: Depends on.

`Default parameters` would be [Chrome 49](https://caniuse.com/?search=default%20parameters) and the game is not on the store, so in practical terms I see no reason why not to make use of it to keep the code concise.

Do you see a reason we would not want to use it?

**Pull request merged**